src/pages/Doyun/Main/components/Feed/Comment.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import './Comment.scss';
+
+export class comment extends Component {
```

Comment:
```suggestion
export class Comment extends Component {
```
컴포넌트 이름은 대문자 사용해주세요!